### DISCUSSION TITLE
Issue закрываются без комментариев либо без решения

### DISCUSSION THREAD

**Discussion by eolme** [VKCOM/VKUI#2487] [General] [answered]
Примеры:

https://github.com/VKCOM/VKUI/issues/1629#issuecomment-1127652764
https://github.com/VKCOM/VKUI/issues/1675#issuecomment-1127698847
https://github.com/VKCOM/VKUI/issues/2442#issuecomment-1127685304

**Comment by pfrankov** [comment]
Все приведённые примеры закрыты с комментариями и с возможными решениями, где это возможно.

**Comment by inomdzhon** [comment, marked as answer]
> #2442 (comment)

тут ты прав, переоткрыл и привязал к #2349

> #1675 (comment)

не выглядит так, что это не влияет на пользовательский опыт, ведь модалка берёт на себя внимания, а задник уходит на второй план

> #1629 (comment)

в прикрепленной ссылке на issue есть информация о `ViewInfinite`, которая судя по запросу, должна решить проблему. Если не решит, то автор issue сможет переоткрыть свой запрос

**Reply by inomdzhon** [reply]
#2442 снова закрыл))

Я добавил ссылку на неё в описание здесь https://github.com/VKCOM/VKUI/issues/2349 и попросил автора подписаться на неё, если ему интересен прогресс по задаче. В данный момент он решил проблему у себя
